Resources/Prototypes/Nyanotrasen/explosion.yml

```diff
@@ -0,0 +1,15 @@
+- type: explosion
+  id: GasSpore
+  damagePerIntensity:
+    types:
+      Caustic: 5
+      Heat: 5
+      Blunt: 2
+      Piercing: 3
+  tileBreakChance: [0, 0.5, 1]
+  tileBreakIntensity: [0, 10, 30]
+  tileBreakRerollReduction: 20
+  lightColor: Orange
+  texturePath: /Textures/Effects/fire.rsi
+  fireStates: 3
```

Comment:
This could probably use a slightly different appearance, maybe something tinted green.

Reply:
I don't even notice it when it explodes tbh, I think the fire frames require a much bigger explosion